README.md

```diff
@@ -103,6 +103,24 @@ Returns the number of subscribers to the current list.
 	$results = $sendy->subcount();
 ```
 
+##createCampaign(array $values)
+
+This method takes an array of `$values` and will creates a campaign (with an option to send it too).
+```php
+	$results = $sendy->createCampaign(array(
+		'from_name' => 'Some Name',
+		'from_email' => 'some@domain',
+		'reply_to' => 'some@domain',
```

Comment:
change the `from_email` and `reply_to` email are missing the `.com` on the end. Change them to be `some@domain.com`